src/molecules/gv-row-expandable.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { css, html, LitElement } from 'lit-element';
+
+/**
+ * Expandable row
+ *
+ * @attr {Boolean} open - When true, opens details by default
+ *
+ */
+export class GvRowExpandable extends LitElement {
+  constructor() {
+    super();
+    this.open = false;
+  }
+
+  static get properties() {
+    return {
+      open: { type: 'boolean' },
+    };
+  }
+
+  static get styles() {
+    return [
+      // language=CSS
+      css`
+        :host {
+          box-sizing: border-box;
+        }
+
+        .row {
+          padding: 0.2rem;
+          cursor: pointer;
+          line-height: 23px;
+          border-bottom: thin solid var(--gv-theme-color);
```

Comment:
I think we have as guideline to always set a default value when using variables `var(--gv-theme-color, #5A7684)`